### PR TITLE
Fix 207

### PR DESCRIPTION
**Pull request by makson96**
This pull request is to address issue #207

The idea is as follows:
After game installation we will create minigalaxy-info.json file in game directory. This file will store game version (from Api) and DLC versions. It can also be used in future to store more data about Game.
If file is present minigalaxy will use it to check if update is available. If file is not present, minigalaxy will fall back to checking gameinfo file.
As it goes to DLCs. We keep compatibility with previous system. If DLC version cannot be determined from minigalaxy-info.json, we will also check minigalaxy-dlc.json. If DLC is present in both files, only minigalaxy-info.json is taken into account.
dlc_backord

This method is applied on all new games, dlcs installation, but backward compatibility is preserved.

I also hope, that code is now more clean and understandable.

Unit tests are adapted to this change and their range is expanded.

### CODE REVIEW

File: tests/test_api.py [hunk 1]
```diff
@@ -9,6 +9,13 @@
 sys.modules['minigalaxy.config'] = m_config
 from minigalaxy.api import Api
 
+API_GET_INFO_TOONSTRUCK = {'downloads': {'installers': [{'id': 'installer_windows_en', 'name': 'Toonstruck', 'os': 'windows', 'language': 'en', 'language_full': 'English', 'version': '1.0', 'total_size': 939524096, 'files': [{'id': 'en1installer0', 'size': 1048576, 'downlink': 'https://api.gog.com/products/1207666633/downlink/installer/en1installer0'}, {'id': 'en1installer1', 'size': 938475520, 'downlink': 'https://api.gog.com/products/1207666633/downlink/installer/en1installer1'}]},
```
Comment (sharkwouter) [inline tests/test_api.py:12]: This is nice, makes the tests a lot cleaner

**Review by sharkwouter — APPROVED** [review]
I'm impressed, this is a pretty clean approach to this. Thanks for the contribution!

I even found an edge case which seems to not even cause any issues. Crystal Caves receives null as version from the API and it doesn't cause an issue.
